.github/workflows/build_and_test.yml

```diff
@@ -1,3 +1,5 @@
+## me fail english? that's unpossible.
+
```

Comment:
that's classified ;0

Reply:
test 1234

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Fault-tolerant

Reply:
test

Reply:
test 6/9/2022

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

Reply:
dadad

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

Reply:
java
Don’t use a for loop with an index (or counter) variable if you can replace it with the enhanced for loop (since Java 5) or forEach (since Java 8). It’s because the index variable is error-prone, as we may alter it incidentally in the loop’s body, or we may starts the index from 1 instead of 0.

📄 [Codejava](https://www.codejava.net/coding/10-java-core-best-practices-every-java-programmer-should-know#LoopIndex)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
If you plot the prices array you may notice that the points of interest are the consecutive valleys and peaks

So, if you will track the growing price and will sell the stocks immediately before the price goe...

📄 [Oleksii Trekhleb](https://github.com/trekhleb/javascript-algorithms/tree/master/src/algorithms/uncategorized/best-time-to-buy-sell-stocks#peak-valley-approach-on)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
test 777

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
test

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient

Reply:
test

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient

Reply:
__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

Reply:
test

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Tags:** Efficient, Not reusable

Reply:
6/28/2022 personal

Reply:
6/28/2022 org

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
test 7/11/2022 readable

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Readable

Reply:
__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Tags:** Readable, Not secure, Efficient, Inelegant, Reusable, Fault-tolerant, Not maintainable

Reply:
test 7/26/2022

Reply:
test 7/26/2022

Reply:
URLs can contain dynamic script content via javascript: protocol URLs. Use validation to assure your links are http: or https: to avoid javascript: URL based script injection. Achieve URL validation using a n...

📄 [snyk](https://snyk.io/blog/10-react-security-best-practices/)

__
[![sema-logo](undefined/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
new test user

__
[![sema-logo](undefined/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome